.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Did you add this file by intention for simplifying for everyone or should this be added to the .gitignore?

Reply:
Sorry... yes, I have it for me in my fork as I was using different machines so I wanted to have it when I clone the repo. 
However, it doesn't have to be here... I can remove it.

Reply:
Ah in that case better add `.vscode` to the `.gitignore` so others can add these files without worry

Reply:
fixed in 4377f3d